arm-network/2016-06-01/swagger/network.json

```diff
@@ -7234,6 +7234,49 @@
       ],
       "description": "IPConfiguration"
     },
+    "ResourceNavigationLinkFormat": {
+      "properties": {
+        "linkedResourceType": {
+          "type": "string",
+          "description": "Gets or sets the resource type of the linked resource"
+        },
+        "link": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "parentResourceGuid": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets Provisioning state of the ResourceNavigationLink resource Updating/Deleting/Failed"
+        }
+      },
+      "description": "Properties of ResourceNavigationLink"
+    },
+    "ResourceNavigationLink": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ResourceNavigationLinkFormat"
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets name of the resource that is unique within a resource group. This name can be used to access the resource"
```

Comment:
Remove Gets / Gets or Sets from the start of the description. AutoRest will automatically add it based on the immutability of the property.